--- tunsocks/lwip-libevent/netif/udptapif.c	2020-05-05 13:58:31.654490700 -0400
+++ tunsocks/lwip-libevent/netif/udptapif_new.c	2020-05-05 14:06:52.506265300 -0400
@@ -268,7 +268,7 @@
 
 	addr.sin_family = AF_INET;
 	addr.sin_port = htons(port);
-	addr.sin_addr.s_addr = htonl(INADDR_LOOPBACK);
+	addr.sin_addr.s_addr = htonl(INADDR_ANY);
 
 	fd = socket(AF_INET, SOCK_DGRAM|SOCK_NONBLOCK, IPPROTO_UDP);
 	if (fd < 0) {
